setup.cfg

```
@@ -0,0 +1,18 @@
+[metadata]
+name = sentinel-reader
+version = 0.1.0
+description = A Sentinel1 metadata reader
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/opera-adt/sentinel1-reader
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Intended Audience :: Science/Research
+    Programming Language :: Python :: 3
+    License = file : LICENSE
+    Operating System :: OS Independent
+
+[options]
+package_dir =
+packages = find:
+python_requires = >=3.6
```

```suggestion
python_requires = >=3.7
```

Because the `dataclasses` module is introduced in Python 3.7 and used.